client-app/src/examples/todo/TodoService.js

```diff
@@ -0,0 +1,95 @@
+import {HoistService, XH} from '@xh/hoist/core';
+import {reject} from 'lodash';
+import {LocalDate} from '@xh/hoist/utils/datetime';
+import {isString} from 'lodash';
+
+/**
+ * Service to manage fetching, updating and persisting tasks.
+ * Tasks are persisted for each user using the Hoist preference system.
+ */
+export class TodoService extends HoistService {
+
+    /** @returns {Promise<TaskItem[]>} */
+    async getTasksAsync() {
+        return XH.getPref('todoApp').map(it => {
+            const dueDate = isString(it.dueDate) ? LocalDate.get(it.dueDate) : null,
```

Comment:
We should just make LocalDate.get() nil safe.  There must be a lot of code like this around.  I will file a ticket.